release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3955,3 +3955,32 @@ $ oc adm release info 4.13.36 --pullspecs
 [id="ocp-4-13-36-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-37"]
+=== RHBA-2024:1200 - {product-title} 4.13.37 bug fix
+
+Issued: 2024-03-13
+
+{product-title} release 4.13.37 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1200[RHBA-2024:1200] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.37 --pullspecs
+----
+
+[id="ocp-4-13-37-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `manila-csi-driver-controller-metrics` service had empty endpoints because an incorrect name for the app selector. With this release the app selector name is changed to `openstack-manila-csi` and the issue is fixed.(link:https://issues.redhat.com/browse/OCPBUGS-26224[*OCPBUGS-26224*])
```

Comment:
```suggestion
* Previously, the `manila-csi-driver-controller-metrics` service had empty endpoints because an incorrect name for the app selector. With this release, the app selector name is changed to `openstack-manila-csi`, and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-26224[*OCPBUGS-26224*])
```